app/controllers/users/utils.js

```diff
@@ -0,0 +1,20 @@
+const bcrypt = require('bcryptjs');
```

Comment:
I think that the best place for this type of logic is the helpers https://github.com/Wolox/tech-guides/blob/master/nodejs/docs/node-standard-and-best-practices.md#24--helpers